config/config.go

```diff
@@ -28,12 +28,13 @@ import (
 )
 
 type System struct {
-	Name     string                    `yaml:"name"`
-	Aliases  []string                  `yaml:"aliases,flow,omitempty"`
-	Overlays []string                  `yaml:"overlays,omitempty,flow"`
-	Master   string                    `yaml:"master"`
-	Workers  []string                  `yaml:"workers,flow,omitempty"`
-	Rabbits  map[string]map[int]string `yaml:"rabbits,flow"`
+	Name      string                    `yaml:"name"`
+	Aliases   []string                  `yaml:"aliases,flow,omitempty"`
+	Overlays  []string                  `yaml:"overlays,omitempty,flow"`
+	Master    string                    `yaml:"master"`
```

Comment:
I think we can have more than 1 master

Reply:
Master isn't even used, so we could get rid of it (in another PR).

Reply:
I can remove it.